packages/devtools_app/lib/src/screens/deep_link_validation/select_project_view.dart

```diff
@@ -26,13 +29,20 @@ class SelectProjectView extends StatefulWidget {
 class _SelectProjectViewState extends State<SelectProjectView>
     with ProvidedControllerMixin<DeepLinksController, SelectProjectView> {
   bool _retrievingFlutterProject = false;
+  String? packageDirectoryForMainIsolate;
 
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
     if (!initController()) return;
   }
 
+  @override
+  void initState() {
+    super.initState();
+    unawaited(_getPackageDirectoryForMainIsolate());
```

Comment:
we should only do this if we are connected to an app.

Reply:
updated